src/Command/PaymentSplitCommand.php

```diff
@@ -155,4 +158,13 @@ private function dispatchTransfers(array $transfers): void
             }
         }
     }
+
+    private function getServiceOrdersPendingDebits(array $orders): array
+    {
+        if (count($orders) > 0) {
```

Comment:
There's no way `count($orders)` isn't > 0, you can remove this condition. 
Instead, I would add `$orderType` to the parameters and check `$orderType == MiraklClient::ORDER_TYPE_SERVICE` here instead of lines 100 and 137.

Reply:
It should be fine now.